lib/src/chart/scatter_chart/scatter_chart_data.dart

```diff
@@ -338,26 +340,25 @@ class ScatterTouchResponse extends BaseTouchResponse with EquatableMixin {
   /// in which spot (of [ScatterChartData.scatterSpots]) touch happened.
   ///
   /// [touchInput] is the type of happened touch.
-  ScatterTouchResponse(FlTouchInput touchInput,
+  ScatterTouchResponse(
```

Comment:
Please revert all formatting changes.
Let me find a way to prevent different formats.
but it is not good to have 22 changes for adding one property.
Thanks!

Reply:
All of these changes became worse as soon as I started working off of dev. Not sure what happened but they appear from the merge, not from my own diff.

Reply:
Are you using dartfmt?